src/rpc_api/data_types.rs

```diff
@@ -624,6 +625,18 @@ impl From<fil_actor_interface::miner::SectorOnChainInfo> for SectorOnChainInfo {
 
 lotus_json_with_self!(SectorOnChainInfo);
 
+#[derive(Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ApiDeadline {
+    #[serde(with = "crate::lotus_json")]
+    #[serde(rename = "PostSubmissions")]
```

Comment:
This is already done by `#[serde(rename_all = "PascalCase")]`, right?
```suggestion
```

Reply:
yes right